Feat(brand): add brand resource (#3147)Co-authored-by: Shanmughapriyan S <priyanshan03@gmail.com> Co-authored-by: Claudio Wunder <cwunder@gnome.org> (#3147)
nodejs/nodejs.dev: content/about/branding.en.md

---
title: branding
displayTitle: Branding of Node.js
description: 'Branding | Node.js'

> authors: AugustinMauroy

Reviewer: I think it's unfair to remove the previous authors, as they also contributed to part of this file.

Author: This is an oversight because I recreated an empty file

Author: it's add 🎉!

---
title: branding
displayTitle: Branding of Node.js
description: 'Branding | Node.js'
authors: AugustinMauroy
category: about
---

Please review the [trademark policy](https://trademark-policy.openjsf.org/) for information about permissible use of Node.js® logos and marks.

## Color

> The following colors are the color palette for the nodejs website. And should be used for other documents.

Reviewer 2: ```suggestion
The following colors are the color palette for the Node.js website. And should be used for other documents.
```

### Brand colors

![Brand colors](/static/images/brand/colors/brand-colors.svg)

### Neutral colors

> 

Reviewer 2: ```suggestion
```

![Neutral colors](/static/images/brand/colors/neutral-colors.svg)

### Status colors

**Danger**

![Danger colors](/static/images/brand/colors/status-danger.svg)

**Warning**

![Warning colors](/static/images/brand/colors/status-warning.svg)

**Info**

![Info colors](/static/images/brand/colors/status-info.svg)

### Accent colors

**Accent 1**

![Accent 1 colors](/static/images/brand/colors/accent-1.svg)

**Accent 2**

![Accent 2 colors](/static/images/brand/colors/accent-2.svg)

## Font

**Font Name**: Open Sans

**Download**: [www.google.com/fonts/specimen/Open+Sans](https://www.google.com/fonts/specimen/Open+Sans)

## Nodejs logos

### Logos with JS hexagon at right

<table>
    <!-- logo dark -->
    <tr>
        <td><a href="/static/images/brand/logos-js-right/dark-no-background.svg">Nodejs logo dark without background</a></td>

>         <td><a href="/static/images/brand/logos-js-right/dark-background.svg">Nodejs logo dark with background</a></td>

Reviewer 2: Change in other occurences too
```suggestion
        <td><a href="/static/images/brand/logos-js-right/dark-background.svg">Node.js logo dark with background</a></td>
```

        <td><a href="/static/images/brand/logos-js-bottom/light-background.svg" download><img src="/static/images/brand/logos-js-bottom/light-background.svg" class="with-bg"></a></td>
    <tr>
    <!-- logo white -->
    <tr>
        <td><a href="/static/images/brand/logos-js-bottom/white-no-background.svg">Nodejs logo white without background</a></td>
        <td><a href="/static/images/brand/logos-js-bottom/white-background.svg">Nodejs logo white with background</a></td>
    </tr>
    <tr>
        <td><a href="/static/images/brand/logos-js-bottom/white-background.svg" download><img src="/static/images/brand/logos-js-bottom/white-no-background.svg" class="with-bg"></a></td>
        <td><a href="/static/images/brand/logos-js-bottom/white-background.svg" download><img src="/static/images/brand/logos-js-bottom/white-background.svg" class="with-bg"></a></td>
    <tr>
</table>

## Assets

### Hexagon

**Green hexagon**

![Green hexagon](/static/images/brand/hexagon/green.svg)

**White hexagon**

![White hexagon](/static/images/brand/hexagon/white.svg)

**Black hexagon**

![Black hexagon](/static/images/brand/hexagon/black.svg)

> ### Js form nodejs logo

Reviewer 2: ```suggestion
### JS from Node.js logo
```

**Green JS**

![Green JS](/static/images/brand/hexagon/js-green.svg)

**White JS**

> <img src="/static/images/brand/hexagon/js-white.svg" alt="White JS" class="with-bg">

Author: I use class to putt an background because the SVG doesn't have background so in light mode we don't see the render. (you can test in local)
